lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,24 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
```

Comment:
Do you need the signature to be `add_instance`?
The object name is `Instance` so maybe just `add` is enough?

Another point is that perhaps this should be the create method of the `Instance` collection?